zebra/if_netlink.c

```diff
@@ -770,6 +771,19 @@ int interface_lookup_netlink(struct zebra_ns *zns)
 	return 0;
 }
 
+static int interface_lookup_netlink_ipv6(struct interface *ifp, struct zebra_ns *zns)
```

Comment:
let's pass in startup so that if this function is ever added else where it can do the right thing.

Reply:
ok